### PR TITLE
fix(calling): extend eventing class on ILine interface 

### PR DESCRIPTION
**Pull request by Kesari3008**
# COMPLETES #[SPARK-479560](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-479560)

## This pull request addresses
There were compilation errors on the ILine interface due to which we weren't able to receive events on line object.

## by making the following changes
Addition of extending eventing class on ILine interface and small corrections.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested
I ran the build and unit tests locally to check everything was passing after the changes.

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/calling/src/CallingClient/line/index.ts [hunk 1]
```diff
@@ -43,13 +43,11 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
...
   public mobiusDeviceId?: string;
 
-  private mobiusUri?: string;
-
   public phoneNumber?: string;
 
   public extension?: string;
 
-  public sipAddresses?: string[];
+  public sipAddresses: string[] = [];
```
Comment (sreenara) [inline packages/calling/src/CallingClient/line/index.ts:50]: Why was this change needed?

**Review by sreenara — CHANGES_REQUESTED** [review]
Please complete the PR description with necessary details:
1. Why the PR was needed
2. Check the bug fix/feature box
3. Tests done

File: packages/calling/src/CallingClient/line/index.ts (hunk 1 above)
Comment (Kesari3008) [reply packages/calling/src/CallingClient/line/index.ts:50]: As discussed, these changes is needed because this property is undefined in the web client which is giving compilation errors.

**Review by sreenara — APPROVED** [review]
